### PR TITLE
feat(client): 65 file structuring

### PR DESCRIPTION
**Pull request by CodenameLiam**

### CODE REVIEW

**Review by Bambanah — APPROVED** [review]

**Review by mvngmnd — APPROVED** [review]

File: client/tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,15 @@
...
 		"jsx": "preserve",
-		"incremental": true
+		"incremental": true,
+		"baseUrl": ".",
+		"paths": {
+			"api/*": ["api/*"],
+			"components/*": ["components/*"],
+			"hooks/*": ["hooks/*"],
+			"utils/*": ["utils/*"],
+			"styles/*": ["styles/*"]
```
Comment (zachnicoll) [inline client/tsconfig.json:23]: Make sure these aliases reflect the actual folders in out repo, e.g. screens folder.

File: client/__mocks__/api.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { createServer, Factory, Model } from 'miragejs';
```
Comment (zachnicoll) [inline client/__mocks__/api.ts:1]: Rebase so these files aren't seen in this PR.

**Review by zachnicoll — CHANGES_REQUESTED** [review]

File: client/tsconfig.json (hunk 1 above)
Comment (zachnicoll) [reply client/tsconfig.json:23]: Actually I think Audi said they were called "templates"

File: client/__mocks__/api.ts (hunk 2 above)
Comment (CodenameLiam) [reply client/__mocks__/api.ts:1]: Never merged mirageJS stuff

Comment (zachnicoll) [reply client/__mocks__/api.ts:1]: okie

**Review by zachnicoll — APPROVED** [review]
🚪

**Pull request merged**